src/services/createRouter.ts

```diff
@@ -192,6 +200,7 @@ export function createRouter<const T extends Routes>(routes: T, options: RouterO
     go: history.go,
     install,
     initialized,
+    isExternal,
```

Comment:
Do we need to expose this?

Reply:
I think it's valuable enough, what do you think?

Reply:
I don't feel strongly. But it's one more thing we have to support publicly.

Reply:
Do you have a good way to hide it? Are you suggesting we just drop it from the type?